code/sql/schema.sql

```diff
@@ -0,0 +1,95 @@
+CREATE SCHEMA waze;
+
+CREATE TABLE waze.jams 
+(
+  "id"                  BIGINT PRIMARY KEY NOT NULL,
+  "uuid"                VARCHAR[500] NOT NULL,
+  "pub_millis"          BIGINT NOT NULL,
+  "start_node"          VARCHAR[500],
+  "end_node"            VARCHAR[500],
+  "road_type"           INTEGER,
+  "street"              VARCHAR[500],
+  "city"                VARCHAR[500],
+  "country"             VARCHAR[500],
+  "delay"               INTEGER,
+  "speed"               float4,
+  "length"              INTEGER,
+  "turn_type"           VARCHAR[500],
+  "level"               INTEGER,
+  "blocking_alert_id"   BIGINT
```

Comment:
is this something we want in the jams table?

Reply:
I think we can keep it there since Waze does.  Actually it looks like it should be a varchar since it's a UUID.  I'l make the edit.